src/library/simplified/extension/domain.js

```diff
@@ -2,8 +2,14 @@ b['lob.domain'] = '';
 if (typeof b['domain'] != "undefined")
 {
     if (b['domain'].indexOf("MULTIITEM") !== -1) {
```

Comment:
@j-ibrahim I can't seem to get "domain" set to "MULTIITEM" to test your changes. I did a FHC, HC and FC searches and "domain" should have been set to "MULTIITEM", but it was not. Therefore, I couldn't move forward of verifying you changes. Is there an AB test that I need to set?